spec/rspec/rails/example/system_example_group_spec.rb

```diff
@@ -16,6 +16,17 @@ module RSpec::Rails
             expect(example.send(:method_name)).to start_with('method_name')
           end
         end
+
+        it 'slices long method name' do
+          group = RSpec::Core::ExampleGroup.describe ActionPack do
+            include SystemExampleGroup
+          end
+
+          example = group.new
+          example_class_mock = double('name' => 'あ'*100)
```

Comment:
Name is generated from the description class / string, set it to a long generated string instead.